app/views/auth.html

```diff
@@ -12,8 +12,9 @@
 
     <div class="loginFields">
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
+      <br>
```

Comment:
Adding the break tags makes it look better, however I would prefer to use css to add spacing for the input fields. Remove the br tags and check the stackoverflow links.

Check this link at stackoverflow http://stackoverflow.com/questions/16379187/spacing-between-form-fields and http://stackoverflow.com/questions/1896567/how-can-i-add-spaces-between-two-input-lines-using-css

Lets try adding this to the css file. Lets see if 10px will look correctly.
input[type="text"] {margin: 0 0 10px 0;}
input[type="password"] {margin: 0 0 10px 0;}
If 10px is too large we can decrease it. Then everything will be all set! Once done, just commit and push your changes back up and I will merge it. 

Thanks!